server/http/oauth2_complete_test.go

```diff
@@ -0,0 +1,270 @@
+package http
```

Comment:
0/5 I have not looked at every detail yet, but I wonder if the tests can be made 100% external to the package and moved into a sub-package (`test_http`?), but only if it doesn't create hurdles in testing.

Reply:
I can work on doing that. I assume if that works without too many hurdles we should the same thing for the tests in the `msgraph` package?